protobuf/pg_query.proto

```diff
@@ -1272,7 +1272,7 @@ message RTEPermissionInfo
 {
   uint32 relid = 1 [json_name="relid"];
   bool inh = 2 [json_name="inh"];
-  int64 required_perms = 3 [json_name="requiredPerms"];
+  uint64 required_perms = 3 [json_name="requiredPerms"];
```

Comment:
As noted in the commit message, this is okay to do only because this struct isn't actually used in raw parse trees. Otherwise this would be an API break (which may be okay in the context of how we use these protobufs, but at least we should talk about doing this outside of major releases).